File: 351001/Orlovich/laba_2/Domain/Models/Issue.cs
using System.Xml.Linq;

namespace Domain.Models;

// Issue
public class Issue: IIdEntity
{
    public long id { get; set; }
    public long authorId  { get; set; }
    public string title { get; set; }
    public string content { get; set; }
    public DateTime created { get; set; }
    public DateTime modified { get; set; }
    public Author author { get; set; }
    public ICollection<Comment> comments { get; set; }
    public ICollection<Marker> markers { get; set; }
}

File: 351002/Ilian_Bukhovets/internal/domain/issue.go
package domain

// Issue представляет задачу/проблему
type Issue struct {
	BaseEntity
	Title      string  `json:"title"`
	Content    string  `json:"content"`
	AuthorID   int64   `json:"authorId"`
	MarkerIDs  []int64 `json:"markerIds"`
}

